conf/api.profile.conf.js

```diff
@@ -1,13 +1,13 @@
-var profile = require('./profile.conf').config;
+var baseProfile = require('./integration.profile.conf').config;
 
-profile.matchers = profile.matchers.concat(
+baseProfile.matchers = [
   {name: './api/toHaveHttpBody'},
   {name: './api/toHaveHttpHeader'},
   {name: './api/body'}
-);
+];
 
-profile.plugins.push({name: '../src/api/requestPlugin'});
```

Comment:
augment the plugins and matchers, don't replace them

Reply:
We would need more code to  handle the missing array case. I plan to handle it generically in an upcoming  profile inheritance feature, this is more like a fix to immediate issues in api profile.

Reply:
wouldn't it be just a check for existence and array type?

Reply:
yes, it always starts with "just ...", we will do it correctly but not in the scope of this task.